shell/probe/raindrop.ps1

```diff
@@ -33,13 +33,24 @@ while ($true) {
             "dat" = $dat
             "version" = "2.1"
         } -UseBasicParsing -MaximumRedirection 0 -ErrorAction SilentlyContinue
-        
+
         $uuid = $task.content -replace ".*?([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}).*", '$1'
         $auth = $task.content -replace '^[^/]*//([^/]*)/.*', '$1'
 
         if ($uuid -and $auth -eq $ca) {
             Invoke-WebRequest -Uri $task.content -OutFile (New-Item -path "$dir\$uuid.exe" -Force ) -UseBasicParsing
-            $code = Execute "$dir\$uuid.exe"
+            $stdoutTempFile = New-TemporaryFile
+            $stderrTempFile = New-TemporaryFile
+            $code = Execute "$dir\$uuid.exe" $stdoutTempFile $stderrTempFile
+            $stdout = Get-Content -Path $stdoutTempFile
+            $stdout = if ($stdout) { [string]::Join("; ", $stdout) } else { "" }
+            $stderr = Get-Content -Path $stderrTempFile
+            $stderr = if ($stderr) { [string]::Join("; ", $stderr) } else { "" }
+            if ($stdout -or $stderr) {
+                Write-Output "${stdout};${stderr}"
+            }
+            Remove-Item -Path $stdoutTempFile
+            Remove-Item -Path $stderrTempFile
```

Comment:
Do we need to catch these deletes so we report even if we can't delete the file?

Reply:
@keitharobertson is there a change accompanying this thumbs up emoji ha

but also, can we just remove these two lines and wait until the test cycle is complete and $dir itself is deleted?

Reply:
oops 🤦 I wrote the code, but then accidentally reverted it. But yes you're right I can just let them get deleted with dir.

Reply:
I got rid of these completely. Will get deleted with the dir deletion